timemachine/potentials/bonded_stable.py

```diff
@@ -43,7 +43,7 @@ def harmonic_angle_stable(conf, params, angle_idxs, cos_angles=True):
     vij = ci - cj
     vkj = ck - cj
 
-    top = jnp.sum(vij * vkj, -1) + eps ** 2
+    top = jnp.sum(vij * vkj, -1)
```

Comment:
The eps^2 term here was unnecessary for stability and introduced an asymmetry biasing the numerator in the positive direction.